ci/scripts/bootstrap.sh

```diff
@@ -16,6 +16,7 @@ elif [[ "${BINARY_TYPE}" == "exe" ]]; then
     source ci/scripts/exe/bootstrap_envvars.sh
 
     echo "--> Install Software via Choco"
+    powershell Install-WindowsFeature Net-Framework-Core # DotNet 3.5 is broken on chocolatey
```

Comment:
True for the CI too?

Reply:
yes, it's necessary ... installation of wixtoolkit failed the same way on CI as locally, this fixes it